README.md

```diff
@@ -2,6 +2,7 @@
 
 
 
+## 
```

Comment:
What is this line doing? Do we need this line in the markdown?

Reply:
Not at all, The branch name shouldn't be at the "Read me" I'm not sure how i added it there, it was probably by mistake.

Reply:
All good, thanks for removing those references! Maybe we had a merge conflict at some point? Either way, looks good now. Thanks for taking care of that.